reactfire/auth/index.tsx

```diff
@@ -42,13 +43,50 @@ export function useUser<T = unknown>(
   );
 }
 
+export function useIdTokenResult(user: User, forceRefresh: boolean = false) {
+  if (!user) {
+    throw new Error('you must provide a user');
+  }
+
+  const idToken$ = from(user.getIdTokenResult(forceRefresh));
+
+  return useObservable(idToken$, `${user.uid}-claims`);
+}
+
 export interface AuthCheckProps {
   auth?: auth.Auth;
   fallback: React.ReactNode;
   children: React.ReactNode;
   requiredClaims?: Object;
 }
 
+export interface ClaimsCheckProps {
+  user: User;
+  fallback: React.ReactNode;
+  children: React.ReactNode;
+  requiredClaims?: Object;
+}
+
+export function ClaimsCheck({ user, fallback, children, requiredClaims }) {
+  const { claims } = useIdTokenResult(user, false);
```

Comment:
@davideast do you think we should [`forceRefresh`](https://firebase.google.com/docs/reference/js/firebase.User.html#getidtokenresult) by default in this component?